[32mINFO    [0m my_log:test_login.py:43 开始调用登录接口,执行2--账号密码错误用例
[32mINFO    [0m my_log:Request_Tools.py:16 请求加载进来了request_mode:<post>  url:<http://47.100.92.21:8080/colatest/colamember/login.do>  headers:<None>  json:<{'accountPWD': '222222', 'phoneNumber': '13333333333'}>  data:<None
[32mINFO    [0m my_log:Request_Tools.py:27 请求结束了,响应信息：<{'message': '登录失败,账号或者密码错误,请重新填写账号和密码!', 'status': '20003'}>
[32mINFO    [0m my_log:test_login.py:45 调用登录接口完成,结果：{'message': '登录失败,账号或者密码错误,请重新填写账号和密码!', 'status': '20003'}
[32mINFO    [0m my_log:test_login.py:50 准备比对结果，实际结果结果《{'message': '登录失败,账号或者密码错误,请重新填写账号和密码!', 'status': '20003'}》,预期结果：{'message': '登录失败,账号或者密码错误,请重新填写账号和密码!', 'status': '20003'}
[32mINFO    [0m my_log:test_login.py:52 测试用例2--账号密码错误通过